react-carousel-component/my-app/src/Container.js

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect } from 'react';
+import './Container.css'
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ */
+export default function Container({ items }) {
+  const [current, setCurrent] = useState(0);
+
+  useEffect(() => {
+    const timerID = setInterval(() => {
+      setCurrent((current + 1) % items.length)
```

Comment:
This usually causes closure problems, though maybe it doesn't because you are running this on every render.

This best way to do this to avoid closure problems is to use the function version of `setCurrent`:
`setCurrent((prev) => (prev + 1) % items.length);`